poetry.lock

```diff
@@ -779,144 +794,183 @@ cron = ["capturer (>=2.4)"]
+python-versions = ">=3.8"
 files = [
-    {file = "comm-0.1.4-py3-none-any.whl", hash = "sha256:6d52794cba11b36ed9860999cd10fd02d6b2eac177068fdd585e1e2f8a96e67a"},
-    {file = "comm-0.1.4.tar.gz", hash = "sha256:354e40a59c9dd6db50c5cc6b4acc887d82e9603787f83b68c01a80a923984d15"},
+    {file = "comm-0.2.0-py3-none-any.whl", hash = "sha256:2da8d9ebb8dd7bfc247adaff99f24dce705638a8042b85cb995066793e391001"},
+    {file = "comm-0.2.0.tar.gz", hash = "sha256:a517ea2ca28931c7007a7a99c562a0fa5883cfb48963140cf642c41c948498be"},
 ]
 
 [package.dependencies]
 traitlets = ">=4"
 
 [package.extras]
-lint = ["black (>=22.6.0)", "mdformat (>0.7)", "mdformat-gfm (>=0.3.5)", "ruff (>=0.0.156)"]
 test = ["pytest"]
-typing = ["mypy (>=0.990)"]
+
+[[package]]
+name = "concrete-python"
+version = "2.5.0rc2"
+description = "A state-of-the-art homomorphic encryption framework"
+optional = false
+python-versions = ">=3.8"
+files = [
+    {file = "concrete_python-2.5.0rc2-cp310-cp310-macosx_11_0_arm64.whl", hash = "sha256:8c8605633ecd43ad8b749ff85074227f2345f9d46c752fcb8972cd32ab84702e"},
+    {file = "concrete_python-2.5.0rc2-cp310-cp310-macosx_11_0_x86_64.whl", hash = "sha256:6a1c8a9bfc71dc9b72709003255ec8d437fa9912d36414a552dfd4f5008095d3"},
+    {file = "concrete_python-2.5.0rc2-cp310-cp310-manylinux_2_28_x86_64.whl", hash = "sha256:1cd681db71afecc3f3592fb20ec87d84c3ad4b95e12f7db8a28856960eede185"},
+    {file = "concrete_python-2.5.0rc2-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:c15c692bd58c9af1c9ed636683024adba3a6aad8224c95104fc55b24dc802ef0"},
+    {file = "concrete_python-2.5.0rc2-cp311-cp311-macosx_11_0_x86_64.whl", hash = "sha256:d3f56ec30db9391c49d4b7288d6b1eb6f3b75bca345a485cfba95ad00dd8ecca"},
+    {file = "concrete_python-2.5.0rc2-cp311-cp311-manylinux_2_28_x86_64.whl", hash = "sha256:44eaaa7c14cfd8208daaa775e2a587caf774f4cf771c702b6a1e7778fa5b5da7"},
+    {file = "concrete_python-2.5.0rc2-cp38-cp38-macosx_11_0_arm64.whl", hash = "sha256:d4546af6f58e0b565290f96e6efdfbbc2ae2f25649063aafe19c72ac337681c9"},
+    {file = "concrete_python-2.5.0rc2-cp38-cp38-macosx_11_0_x86_64.whl", hash = "sha256:372fad620f0063e1bf46fb2c5c8e6dd4f1a792b38a7eab839090d911ca7bc145"},
+    {file = "concrete_python-2.5.0rc2-cp38-cp38-manylinux_2_28_x86_64.whl", hash = "sha256:6d73446e665a4d659efc95cbf06e504582e400a024ae8451c8a47df31ed91a7b"},
+    {file = "concrete_python-2.5.0rc2-cp39-cp39-macosx_11_0_arm64.whl", hash = "sha256:41aab449f981e6eec904156633dd1cf6628fc1b62c23d6e2d59cab964320db01"},
+    {file = "concrete_python-2.5.0rc2-cp39-cp39-macosx_11_0_x86_64.whl", hash = "sha256:25360c5b07939c9b700185fafa042d2134e7bcec1cb4baff53c7885defa51f36"},
+    {file = "concrete_python-2.5.0rc2-cp39-cp39-manylinux_2_28_x86_64.whl", hash = "sha256:cae76016f85a5dc07bc6da44650107e9874326b195e1f033cc296c278bb80306"},
+]
+
+[package.dependencies]
+importlib-resources = ">=6.1"
+networkx = ">=2.6"
+numpy = ">=1.23"
+scipy = ">=1.10"
+torch = ">=1.13"
+z3-solver = "4.12.2.0"
```

Comment:
This line was modified manually, because poetry generates `4.12.2` as version number instead of `4.12.2.0` for an unknown reason.

This then breaks `poetry install`.
To keep in mind for further updates.
I'll open an issue to poetry.

Reply:
Not sure if this is sustainable. Every time there is an update, you'll need to manually change it :thinking:

Reply:
Yes I know, this is a workaround until poetry fixes that.